extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -210,7 +268,8 @@ private void handleJaxbFile(Path p, BuildProducer<NativeImageResourceBuildItem>
             BuildProducer<ReflectiveClassBuildItem> reflectiveClass) {
         try {
             String path = p.toAbsolutePath().toString().substring(1);
-            String pkg = p.toAbsolutePath().getParent().toString().substring(1).replace("/", ".") + ".";
+            String pkg = p.toAbsolutePath().getParent().toString().substring(1)
+                    .replace(FileSystems.getDefault().getSeparator(), ".") + ".";
```

Comment:
maybe use File.separatorChar or separator.
https://docs.oracle.com/javase/8/docs/api/java/io/File.html#separator

Reply:
Yes, that would be better.

Reply:
Also, we need a test that shows it's broken and now works on Windows.

Reply:
@gsmet you mean a unit test because there are none in jaxb extension currently. or test from user ?
if unit test , in deployment or in integration test ?

Reply:
I have add a test in integration test.